### ISSUE TITLE
Why was the graph with active (and total) cases removed?

### ISSUE THREAD

**Issue by sonnichjensen** [okestonia/koroonakaart#272]
I am missing the graph with the active cases count.  It was a good view of the situation since the beginning, and a good way to follow situation.

Why was this removed?

I would like to see it back

**Comment by ostankin** [comment]
I came here to ask the same question. The `Total cases` chart used to display stats on Active cases, and also "In hospital" and "On ventilation". I think that stats related to the load of hospitals are even more important in the current stage, where vaccination or previous infection barely prevents from (re-)infection.

**Comment by askoseeba** [comment]
Same here. The main chart that I used to visit was the one that showed the current hospitalization, assisted breathing and death rates -- the most important measures why the whole hassle of restrictions at all (in order to keep the load on hospitals down), but that chart that included those measures seems to be gone. Why? Is it possible to bring it back?

**Comment by sonnichjensen** [comment]
Exactly. Current active cases in general, will tell the story for all the time in one single graph. As will the other numbers for whatever ones interest is.
Restoring should be easy, they must have the code present in and old commit.

**Comment by askoseeba** [comment]
Alternative place to get the same information (in a heatmap form with age group granularity) is here: https://biit.cs.ut.ee/covid/#/agegroups?lang=en -- scroll down on that page where you see the "New hospitalisations per age group" visualization.

**Comment by neuroactive** [comment]
Thanks everyone for the feedback, and sorry that some of the information you are looking for is currently unavailable. Some of the charts you referred to have been temporarily removed for maintenance. The fuller explanation of why some charts are currently missing is as follows.

There were some issues previously with how the charts were presenting the data, for example the “Cases” chart was mixing cumulative data on cases and deaths with daily figures for various statistics on hospitalisation as well as the “Active cases estimate”. In general, there was considerable room for improvement in terms of how the charts were structured, so in December I started working on a review of the charts to address these issues.

Unfortunately, as a result of the fact that the data files being downloaded from TEHIK have now grown extremely large, in mid-December the daily update for the site started to fail due to a lack of memory. Rather than implement the performance optimisations needed to get the updates working again on the existing (legacy) codebase, I decided to implement those into the development branch and send that live sooner than planned. That branch already included various performance and other improvements and so enabled the issue with the update to be fixed quicker. The downside was that the detailed information on deaths and hospitalisations was temporarily removed as work on those was incomplete. I am working to restore that information, however, and expect to have new charts for those live by the end of this week.

The “Active cases estimate” is a different issue. This statistic seems potentially problematic to me for several reasons. One is the somewhat arbitrary and unscientific nature of how an “active case” has been defined, i.e. it simply includes any confirmed case that has occurred within the past two weeks. However, my main concern is that the wording may give the average user the impression that it represents the total number of people who are currently infected with Covid in Estonia or each county. In reality, that statistic is likely to significantly underestimate the true number of people who are infected due to various factors such as the fact that not all people with Covid seek medical attention, that only PCR test results are taken into account in the statistics, and that there are many mildly sympomatic and asymptomatic cases which are never detected. I do think the Koroonakaart site has a responsibility not to publish information which has the potential to mislead, especially if such information could lead people to believe that they are at less risk than they actually are. Having said that, it’s clear from the comments that the site’s users appreciate having concrete numbers on active cases presented. What I propose to do then is to restore that chart, but to rename it to “Confirmed cases (past 14 days)”, which is arguably more accurate and will hopefully be less likely to give the impression that it represents the total number of currently-infected people. In the meantime, I would point out that the “14-day infection rate per 100,000” chart essentially includes exactly the same data as the active cases estimate, just adjusted by the population, so the trend in the national “active cases estimate” can be seen by looking at that infection rate chart.

**Comment by sonnichjensen** [comment]
Active cases = sum of positive tests for the last 14 days. I know this is not true, but it will give a graph that shows highs and lows, as well as trends. That is all I am looking for.

**Comment by neuroactive** [comment]
@sonnichjensen Yes, but the "Cases per 100,000" chart is also based on the total number of cases for the last 14 days and so should enable you to see the highs, lows, and trends in exactly the same way as the "active cases estimate" chart did. The shape of those two charts should be identical.

**Comment by askoseeba** [comment]
Thank you for shedding some light on the maintenance challenges! You guys are doing awesome work! I completely agree with being cautious on avoiding communicating false messages. For me personally the "active cases" has always meant implicitly "confirmed active cases", as I understand nobody can really know how many people are actually currently carrying active virus. Partially for that same reason I am even not that interested in the "active cases" estimate. Partially, because the other reason is, the whole struggle for the restrictions, lockdowns etc is not because of the amount of active cases, but because of the workload of hospitals. That is the reason the information that shows directly the hospitals workload is the most important one, and fortunately there we have the adequate data available. The trendline of (confirmed) active cases is, if anything, more like a proxy or slight predictor to whether we might have a trouble (overloaded hospitals) ahead in 2 weeks or not. So I am looking forward to see those trend visualizations back up and running. Keep up the good work!

**Comment by sonnichjensen** [comment]
@neuroactive , which makes it easy to choose from either "Cases per 100000" or "Cases count". Data are there I understand

**Comment by ostankin** [comment]
@neuroactive, it looks like your plan to restore hospitalisations and deaths in the "Cases" chart is taking a bit longer than expected. Can you please update us on the new ETA? This data is becoming rather important, because the infection rate can't be compared with previous waves anymore - I would rather compare hospitalisations and ventilations.
